website/content/api-docs/ui.mdx

```diff
@@ -14,7 +14,7 @@ This page lists all known jobs in a paginated, searchable, and sortable table.
 
 | Path       | Produces    |
 | ---------- | ----------- |
-| `/ui/jobs` | `text/html` |
+| `/v1/ui/jobs` | `text/html` |
```

Comment:
We shouldn't have the `/v1` prefix on any of the paths on this particular page because  the web UI doesn't have that. Ex. http://localhost:4646/ui/jobs is the URL that brings you to the jobs page. 

(That page then hits the API with the `/v1` prefix shown on all the other pages in this `api-docs` section.)